helpers.py

```
@@ -217,3 +251,99 @@ def generate_rpc_pass(length):
+    """Return a list of host values from seed-nodes.json.
+
+    Prefers local config/seed-nodes.json. If missing, attempts to download.
+    Returns [] on failure.
+    """
+    try:
+        print(f"Getting seed nodes list from {_SEED_NODES_PATH}...")
+        if os.path.exists(_SEED_NODES_PATH):
+            with open(_SEED_NODES_PATH, "r", encoding="utf-8") as f:
+                try:
+                    data = json.load(f)
+                except Exception as e:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)